core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/ColumnsSelectionDsl.kt

```diff
@@ -422,7 +408,16 @@ public interface ColumnsSelectionDsl<out T> : /* SingleColumn<DataRow<T>> */
      *
      *
      */
-    public interface DslGrammar
+    public interface DslGrammar {
+
+        
+
+        
```

Comment:
is it fine to have so much blank lines?

Reply:
It's the result of `@ExcludeFromSources` which removes some documentation interfaces. The newlines are originally from in between these interfaces.
When we've updated our linter (https://github.com/Kotlin/dataframe/issues/364) I'll turn on the auto-reformat for generated files, so we can fix these cases :)